shared/locales/en-us.json

```diff
@@ -0,0 +1,7 @@
+{
+  "ATTRIBUTE": "attribute",
+  "neo-codap:ATTRIBUTE": "column",
```

Comment:
Rather than mixing strings from multiple applications into a single strings file, my thought would be to support multiple strings files that are merged dynamically, e.g. (conceptually) `strings = Object.assign({}, "strings.json", "neo-codap-strings.json")`, where neo-CODAP strings can override the default strings.